test/e2e/rollout_test.go

```diff
@@ -542,61 +544,11 @@ var _ = Describe("placing wrapped resources using a CRP", Ordered, func() {
 	})
 })
 
-func readDeploymentTestManifest(testDeployment *appv1.Deployment) {
-	By("Read the deployment resource")
-	err := utils.GetObjectFromManifest("resources/test-deployment.yaml", testDeployment)
-	Expect(err).Should(Succeed())
-}
-
-func readDaemonSetTestManifest(testDaemonSet *appv1.DaemonSet) {
-	By("Read the daemonSet resource")
-	err := utils.GetObjectFromManifest("resources/test-daemonset.yaml", testDaemonSet)
-	Expect(err).Should(Succeed())
-}
-
-func readStatefulSetTestManifest(testStatefulSet *appv1.StatefulSet) {
-	By("Read the statefulSet resource")
-	err := utils.GetObjectFromManifest("resources/test-statefulset.yaml", testStatefulSet)
-	Expect(err).Should(Succeed())
-}
-
-func readServiceTestManifest(testService *corev1.Service) {
-	By("Read the service resource")
-	err := utils.GetObjectFromManifest("resources/test-service.yaml", testService)
-	Expect(err).Should(Succeed())
-}
-
-func readJobTestManifest(testManifest *batchv1.Job) {
-	By("Read the job resource")
-	err := utils.GetObjectFromManifest("resources/test-job.yaml", testManifest)
-	Expect(err).Should(Succeed())
-}
-
-func readEnvelopeConfigMapTestManifest(testEnvelopeObj *corev1.ConfigMap) {
-	By("Read testEnvelopConfigMap resource")
-	err := utils.GetObjectFromManifest("resources/test-envelope-object.yaml", testEnvelopeObj)
-	Expect(err).Should(Succeed())
-}
-
 // createWrappedResourcesForRollout creates an enveloped resource on the hub cluster with a workload object for testing purposes.
 func createWrappedResourcesForRollout(testEnvelopeObj *corev1.ConfigMap, obj metav1.Object, kind string, namespace corev1.Namespace) {
```

Comment:
A nit: after the latest refactoring this function feels somewhat redundant to me. Not a blocking issue at all though; feel free to ignore.

Reply:
I tried but found out that the diff makes the code a little bit cleaner.